zoo/analytics/tasks/docker.py

```diff
@@ -1,41 +1,44 @@
+from pathlib import Path
+
 from dockerfile_parse import DockerfileParser
 
-from . import Language, OS
+from . import DockerImage, OS
 from ..models import DependencyType
-from .utils import parse_version
+from .utils import DockerImageId
 
 KNOWN_DEPS = {
     DependencyType.LANG: {"python", "node"},
     DependencyType.OS: {"alpine", "debian"},
 }
 KNOWN_DEBIAN_IMAGES = {"python", "node"}
 
+# TODO: make more clever ways of dealing with Dockerfile paths.
+DOCKERFILE_PATHS = ["/", "/docker/"]
+
 
 def analyze(repository, path):
     """Parse dockerfile for image information about platform."""
-    dockerfile = path / "Dockerfile"
-    if not dockerfile.is_file():
-        return
-    dockerfile.resolve()
-    dfp = DockerfileParser(path=dockerfile.as_posix())
-    for command in dfp.structure:
-        if command["instruction"] != "FROM":
+    for sub_path in DOCKERFILE_PATHS:
```

Comment:
Instead of using `DOCKERFILE_PATHS` I think it could be better to just glob `Dockerfile*`. It's convention to name them like that, and actually with this solution we wouldn't consider `Dockerfile.cypress` for example

Reply:
Makes sense, changed

Reply:
But you haven't pushed it yet, right? I can't see the changes 🤔 😅

Reply:
indeed, pushed just now